boards/posix/native_posix/doc/index.rst

```diff
@@ -516,6 +516,10 @@ The following peripherals are currently provided with this board:
   must be powered down and support Bluetooth Low Energy (i.e. support the
   Bluetooth specification version 4.0 or greater).
 
+**USB controller**:
+  It's possible to use the Virtual USB controller working over USB/IP protocol.
+  More information can be found in :ref:`Testing USB over USP/IP in native_posix`.
```

Comment:
This one ```:ref:`Testing USB over USP/IP in native_posix` ``` is not something you can refer from other docs.
You need to add a reference on it, something like:
(that line is over 80 chars by the way)
```diff
diff --git a/boards/posix/native_posix/doc/index.rst b/boards/posix/native_posix/doc/index.rst
index 2c4c8c1e89..8712b1a0a4 100644
--- a/boards/posix/native_posix/doc/index.rst
+++ b/boards/posix/native_posix/doc/index.rst
@@ -517,8 +517,9 @@ The following peripherals are currently provided with this board:
   Bluetooth specification version 4.0 or greater).
 
 **USB controller**:
-  It's possible to use the Virtual USB controller working over USB/IP protocol.
-  More information can be found in :ref:`Testing USB over USP/IP in native_posix`.
+  It's possible to use the Virtual USB controller working over the USB/IP
+  protocol. More information can be found in
+  :ref:`Testing USB over USP/IP in native_posix <testing_USB_native_posix>`.
 
 **Display driver**:
   A display driver is provided that creates a window on the host machine to
diff --git a/doc/reference/usb/index.rst b/doc/reference/usb/index.rst
index a1a794834d..8760ef468f 100644
--- a/doc/reference/usb/index.rst
+++ b/doc/reference/usb/index.rst
@@ -123,6 +123,8 @@ the vendor requests:
 The class driver waits for the :makevar:`USB_DC_CONFIGURED` device status code
 before transmitting any data.
 
+.. _testing_USB_native_posix:
+
 Testing USB over USP/IP in native_posix
 ***************************************
```

Reply:
Sorry for that, somehow I could not build docs locally, I thought CI will show errors if any.

Reply:
I believe I fixed this now.